dist/templates/cleanup-ovn-cni.conf.j2

```diff
@@ -0,0 +1,2 @@
+r /etc/cni/net.d/10-ovn-kubernetes.conf
+r /etc/origin/openvswitch/conf.db
```

Comment:
The deletion of conf.db only occurs in the case of OpenShift right? We don't want this to happen in just OVN case since folks might be running OVS as a non-container, i.e., OVS servers will be running in the Host instead of a container in the Pod.

Reply:
The issue issue is not container vs host. It is an issue of data in the db that reflects the system state before the reboot and is not valid after the reboot. The OVS maintainers recommend deleting the ovs DB during boot. It will be recreated as part of normal processing.